getting-started/blazor/your-first-chart-using-blazor.md

```diff
@@ -6,88 +6,80 @@ heading: Create a Chart in Blazor using FusionCharts
+* Visual Studio (Used Community Edition 2022 - 17.6.1) on Windows or Mac.
+* Alternatively, you can install [.NET for MacOS](https://learn.microsoft.com/en-us/dotnet/core/install/macos) or [.NET () for Linux](https://learn.microsoft.com/en-us/dotnet/core/install/linux).
 
 **IMPORTANT NOTE:** Install Blazor if not present already.
 
-
 ## Installation 
-
-Follow these simple steps to install Blazor.
-
-1. Clone the code repository to your local machine from [blazor-fusioncharts](https://github.com/fusioncharts/blazor-fusioncharts).
-
-**For Windows**: 
-1. Ensure the following workloads are selected during the Visual Studio installation:
-- ASP.NET and Web Development
-- Azure Development
-- .NET Desktop
-
-2. After installation, open the Visual Studio application and select the “Open a project solution” option. 
-
-3. Navigate and select the ‘BlazorApp1.sln’ file in the extracted folder. 
-
-4. Finally, run the ‘BlazorApp1.sln’ file by clicking the run button in the toolbar. 
-
-**For MacOS and Linux**:
-1. Open the terminal from your project directory and run the ‘dotnet watch’ command.
-
-2. Open a web browser and navigate to the appropriate address to view and interact with your Blazor application.  
-
 ### Setup using NuGet Package
 Follow these steps to install Blazor using the NuGet Package.
 
-1. Open the project inside the Visual Studio app.
+1. Open the project in Visual Studio.
 
-2. On the Settings icon, select “Options”.
+2. Go to `Manage Nuget Packages`, search for `BlazorFusionCharts`, and click the `Add Package` button to install the package.   
 
-3. Find the NuGet Package Manager and select Package Sources from the left panel of the Options window.
+### Installation and Including Dependencies
+To create a new Blazor application:
+1. Remove the FusionCharts package from the following locations:
+- For Windows: `C:\Users\XXX.nuget\packages`
+- For macOS and Linux: `~/.nuget/packages`
 
-4. click the `+` symbol in the top-right corner to add a package.
-- For the Name field, add the name of the package source as `myget.org`.
-- For the Source field, paste the myGet Package link.
-- Click “Update”, and then click “OK”.
+2. Remove any fusionCharts package from the local at the following locations:
+- For Windows: `C:\Users\XXX.nuget\packages`
+- For macOS and Linux: `~/.nuget/packages`
 
-5. Under Tools, click the “NuGet Package Manager” option and select Manage NuGet Packages for Solution.   
+3. Install the latest package from [Nuget.org](https://www.nuget.org/).
 
-6. Change the project source to “myget.org” in the top-right corner. 
+4. Upon successful installation, manually add the `blazor-fusioncharts.js` file from the “C:\Users\XXX.nuget\packages\blazorfusioncharts\1.0.0\staticwebassets” address to the wwwroot folder. 
 
-7. Finally, move the browser tab of the same window, enter the credentials, and install the package "BlazorApp1".
+5. In the `Hosts.cshtml` file, add the following script tags:
```

Comment:
The filename is _Host.cshtml. Please refer the ReadMe: https://github.com/fusioncharts/blazor-fusioncharts/tree/master/BlazorFusionCharts